src/java.base/share/classes/java/util/Locale.java

```diff
@@ -1237,12 +1236,11 @@ public static String[] getISOCountries() {
     }
 
     /**
-     * Returns a {@code Set} of ISO3166 country codes for the specified type.
+     * {@return a {@code Set} of ISO3166 country codes for the specified type}
```

Comment:
This and other instances will have the Returns ... and @return ... differ by a single word. Such as

_Returns a {@code Set} of ISO3166 country..._ 
vs 
_@return a {@code Set} of ISO country..._

{@return is used when the Returns ... and @return ... are identical. 

I want to double check that using {@return for these instances is acceptable (even though they are technically not identical), if not I can revert them.